discovery-provider/alembic/versions/dc7f691adc79_add_aggregate_track_table.py

```diff
@@ -0,0 +1,332 @@
+"""add aggregate track table
+
+Revision ID: dc7f691adc79
+Revises: 8e4dda8255fd
```

Comment:
Maybe you're aware of this already but something to note before merging is we will need to update this revision ID since there have been new alembic migrations and two versions can't have the same down_version. you can update this to the new head when you update this with master.

Reply:
Yup! updated now.